### ISSUE TITLE
[Translation] Add Bengali/Bangla translation

### ISSUE THREAD

**Issue by Oymate** [mathjax/MathJax#2677]
**Is your feature request related to a problem? Please describe.**
I would like mathjax translated to have a better experience.

**Describe the solution you'd like**
I can help translating but I need documentation

**Describe alternatives you've considered**
I can also do it on some other platform than git

**Additional context**

**Comment by zorkow** [comment]
Many thanks for your interest. Can you clarify a bit what exactly you mean by translation.
Are you referring to the a11y component? If so, we would certainly be interested. We are just finishing off Hindi localisation, so another language from the subcontinent would certainly be great.

**Comment by Oymate** [comment]
I believe so, but looking for a translation string I can see it references something not in this repo. So, I need a bit of documentation :)

**Comment by zorkow** [comment]
Yes, the core a11y component is provided by the [Speech Rule Engine](https://github.com/zorkow/speech-rule-engine) which is not part of this repository.  As an example [here is Hindi localisation]  (https://github.com/zorkow/speech-rule-engine/tree/develop/src/mathmaps/hi).
To get started it is probably easiest to have chat first. Let me know when would be a good time for you.

**Comment by Oymate** [comment]
I can start at any moment [Do I need to join a chat or something?]

**Comment by zorkow** [comment]
I will have some time tomorrow to put the initial material together. If you send me your gmail address I can share a folder with you, containing spreadsheets for the localisation. My email is in my profile.

**Comment by Oymate** [comment]
Hey, check your email please :)

**Comment by Oymate** [comment]
@zorkow In the meeting?

**Comment by zorkow** [comment]
@Oymate I am in the meeting. FYI: sococo only works with Chrome. No other browser.
